### DISCUSSION TITLE
Tailwind Typology plugin doesn't change font style of heading tags from default 16px size

### DISCUSSION THREAD

**Discussion by huskyjp** [tailwindlabs/tailwindcss#7265] [Help] [answered]
I am trying to parse MDX file and convert those into React components.
I use TailwindCSS, Nextjs, React - as Tailwind documentation illustrated, I installed tailwind/typology plugin! (below are the setup what I tried)

tailwind.config.js
```
	plugins: [require('@tailwindcss/typography'), require('@tailwindcss/aspect-ratio'), require('tailwindcss-textshadow')]

```

globals.css
```
@tailwind base;
@tailwind components;
@tailwind utilities;

@layer base {
	h1 {
		@apply mb-6 text-3xl font-semibold;
	}
	h2 {
		@apply text-2xl font-semibold;
	}
	p {
		@apply mb-4;
	}
	a {
		@apply text-blue-500 hover:text-blue-400 dark:text-blue-400 dark:hover:text-blue-300;
	}
```

and then in my Blog function I use MDXRemote to render HTML Element.
```
export const Blog = ({ source, frontMatter }: BlogInput) => {
	return (
			<article className="prose">
				<h2>This should have H2 heading style!</h2>
				<div className={s.title}>{frontMatter.title}</div>
				<div>{format(parseISO(frontMatter.date!), 'MMMM dd, yyyy')}</div>
				<div className="">
					<MDXRemote {...source} components={components} />
				</div>
			</article>
	)
}
```

This is my example MDX file.
```
---
title: Example Post
description: '',
date: '2022-01-30'
image: '/images/hello.jpg'
---

# Heading

### This is an example blog post with React components.
```

As you can see from this screenshot, it doesn't render with custom h1 h2 heading font style but static 16px text. (I think this is the default font style by tailwind team).  `Example Post` font style has changed since I directly added css from classname.

[![enter image description here][1]][1]

When I check the font from VSCode Preview extension, I can clearly see the font style has changed.
[![enter image description here][2]][2]

Is there any thing I am missing?

  [1]: https://i.stack.imgur.com/CEi3x.jpg
  [2]: https://i.stack.imgur.com/70gfr.png

**Comment by adamwathan** [comment]
Can you provide a reproduction? Could be your `content` config.

**Comment by huskyjp** [comment, marked as answer]
@adamwathan Just figured out I am overwriting css default setting by declaring `import 'tailwindcss/tailwind.css'`  in my `_app.tsx`. 

I was like this - thanks!
```
import '../styles/globals.css'
import 'tailwindcss/tailwind.css' <- this overrides all configs 🙏
import type { AppProps } from 'next/app'

function MyApp({ Component, pageProps }: AppProps) {

	return (
		<>
			<Component {...pageProps} />
		</>
	)
}

export default MyApp
```
